CHANGES.md

```diff
@@ -2,13 +2,18 @@
 
 ## __NEXT__
 
+### Major changes
+
+* filter: Duplicate entries in `--sequences` will now result in an error. [#1613] (@victorlin)
```

Comment:
[not blocking] consider rewording to "Duplicate header names in the FASTA file (`--sequences`) will now...", as duplicated sequences are just fine.

Reply:
Thanks! Reworded:

https://github.com/nextstrain/augur/blob/da9301c586b594e71d4b491331a566af535e1cd3/CHANGES.md?plain=1#L7